manifests/init.pp

```diff
@@ -22,19 +22,25 @@
   require wget
   require adobe_em6::tools
 
-  if $adobe_em6::params::keystore_source_location != 'UNSET' {
-    file { "${dir_aem_certs}/aem_keystore.jks":
-      ensure  => 'present',
-      source  => "${adobe_em6::params::keystore_source_location}",
-      require => File[ $dir_aem_certs ],
+  $exec_path = ['/bin', '/usr/bin', '/usr/local/bin', '/usr/java/latest/bin/' ]
```

Comment:
@jbornemann  I think you can just set Exec { path => '/bin:/usr/bin' } and get rid of the need for path args on L33 & L42 (per https://docs.puppetlabs.com/puppet/3/reference/lang_defaults.html)